src/reynir_correct/errfinder.py

```diff
@@ -554,11 +554,23 @@ def VillaÍTölu(self, txt: str, variants: str, node: Node) -> AnnotationDict:
         subject = self._node_text(ch0)
         # verb_phrase = self._node_text(children[1])
         number = "eintölu" if "et" in variants else "fleirtölu"
+        # Find the verb
+        ch1node = cast(Node, ch1)
+        vptree: Optional[SimpleTree] = self._simple_tree(ch1node)
+        so: Optional[SimpleTree]  = None
+        if vptree:
+            vp: Optional[SimpleTree] = vptree.first_match("VP >> so")
+            if vp:
+                so = vp.first_match("so")
         # Annotate the verb phrase
         assert ch1 is not None
         start, end = self._node_span(ch1)
+        if so is not None:
+            sostart, soend = so.span
+            end = start + soend
+            start = start+sostart
```

Comment:
Missing spaces here - would be fixed by running the Black formatter

Reply:
Done